doc/build_and_test.md

```
@@ -32,7 +32,7 @@ E2E suite runs the built `sqly` binary rather than linking a library; install it
 separately with the version CONTRIBUTING.md pins, which is the one CI uses:
 
 ```shell
-$ go install github.com/nao1215/atago@v0.18.0
+$ go install github.com/nao1215/atago@v0.20.0
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)